### PR TITLE
[fxnetworks] improve extraction

### PR DESCRIPTION
**Pull request by enigmaquip**
## Please follow the guide below

- You will be asked some questions, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *pull request* (like that [x])
- Use *Preview* tab to see how your *pull request* will actually look like

---

### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [adding new extractor tutorial](https://github.com/rg3/youtube-dl#adding-support-for-a-new-site) and [youtube-dl coding conventions](https://github.com/rg3/youtube-dl#youtube-dl-coding-conventions) sections
- [x] [Searched](https://github.com/rg3/youtube-dl/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests
- [ ] Checked the code with [flake8](https://pypi.python.org/pypi/flake8)

### In order to be accepted and merged into youtube-dl each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [ ] Bug fix
- [x] Improvement
- [ ] New extractor
- [ ] New feature

---

### Description of your *pull request* and other information

Explanation of your *pull request* in arbitrary form goes here. Please make sure the description explains the purpose and effect of your *pull request* and is worded well enough to be understood. Provide as much context and examples as possible.

Adding Series name, episode and season info to the extractor

### CODE REVIEW

File: youtube_dl/extractor/fxnetworks.py [hunk 1]
```diff
@@ -64,6 +64,9 @@ def _real_extract(self, url):
             'id': video_id,
             'title': title,
             'url': smuggle_url(update_url_query(release_url, query), {'force_smil_url': True}),
+            'series': video_data.get('data-show-title'),
+            'episode_number': int_or_none(video_data.get('data-episode')),
+            'season_number': int_or_none(video_data.get('data-season')),
```
Comment (dstftw) [inline youtube_dl/extractor/fxnetworks.py]: Update test.

File: youtube_dl/extractor/fxnetworks.py [hunk 2]
```diff
@@ -13,17 +13,21 @@
 class FXNetworksIE(AdobePassIE):
     _VALID_URL = r'https?://(?:www\.)?(?:fxnetworks|simpsonsworld)\.com/video/(?P<id>\d+)'
     _TESTS = [{
-        'url': 'http://www.fxnetworks.com/video/719841347694',
-        'md5': '1447d4722e42ebca19e5232ab93abb22',
+        'url': 'http://www.fxnetworks.com/video/1046477891799',
```
Comment (dstftw) [inline youtube_dl/extractor/fxnetworks.py]: Test video must be a freely available one.

**Comment by enigmaquip** [comment]
sorry about that, fixed the test vid

**Comment by dstftw** [comment]
No, you didn't. Have you even tried to run it after these changes?

**Comment by enigmaquip** [comment]
I had two different versions on two different computers. I feel dumb. I swear I'm getting passing this time with what I have comitted...
```
python test/test_download.py TestDownload.test_FXNetworks
[FXNetworks] 1032565827847: Downloading webpage
[ThePlatform] dRzwHC_MMqIv: Downloading SMIL data
[ThePlatform] dRzwHC_MMqIv: Checking video URL
[ThePlatform] dRzwHC_MMqIv: Checking video URL
[ThePlatform] dRzwHC_MMqIv: Checking video URL
[ThePlatform] dRzwHC_MMqIv: Checking video URL
[ThePlatform] dRzwHC_MMqIv: Checking video URL
[ThePlatform] dRzwHC_MMqIv: Checking video URL
[ThePlatform] dRzwHC_MMqIv: Downloading JSON metadata
[info] Writing video description metadata as JSON to: test_FXNetworks_dRzwHC_MMqIv.info.json
[debug] Invoking downloader on u'http://fxdlv.fxnetworks.com/FX_Networks/668/523/BET_S2_FIRSTLOOK_SEASONPREMIERE_SEPT14_CLEAN_FXWEB_1033460803894_mp4_video_1280x720_2500000_primary_audio_eng_8.mp4?__gda__=1509228942_3d55a77ce16e404060ef656258ad818d'
[download] Destination: test_FXNetworks_dRzwHC_MMqIv.mp4
[download] 100% of 10.00KiB in 00:00
.
----------------------------------------------------------------------
Ran 1 test in 2.247s

OK
```

**Pull request merged**